addon/kubeflow.v1.2.0/manifest.yaml

```diff
@@ -190,6 +200,22 @@ spec:
         name: manifests
         path: stacks/kubernetes/application/cert-manager
     name: cert-manager
+{{ if and .Variables.EnableTLS .Variables.DomainName }}
+  - kustomizeConfig:
+      parameters:
+      - name: email
+        value: {{ .Cluster.OwnerEmail }}
+      - name: acmeUrl
+        value: https://acme-v02.api.letsencrypt.org/directory
+      - name: namespace
```

Comment:
I see you also add the namespace in the base directly: https://github.com/kubermatic/flowmatic/blob/da72c0aae432c0d7053241f78fe9038bfabd9a88/manifests/istio/ingressgateway-acme-cert/base/kustomization.yaml#L17
so I guess it is not necessarily to add namespace parameter here again

Reply:
yes, we can remove this one

Reply:
done